src/lib/translate-link.ts

```diff
@@ -28,7 +52,11 @@ const URL_TRANSLATIONS: Map< string, Partial< Record< SupportedLocale, string >
 ] );
 
 export function translateLink( locale: SupportedLocale, url: string ): string {
-	const translation = URL_TRANSLATIONS.get( url )?.[ locale ];
```

Comment:
Following the single responsibility principle, and considering that the two implementations differ significantly, I think it would make sense to have two separate files, one for `getDocsLink` and another for `getBlogLink`. Alternatively, have `getDocsLink` which dynamically changes the language, and another one, e.g. `getLink` that uses the mapping array.

Reply:
Good point. So I opted to share the same logic and architecture. Now the translatable links use a dict/object. Types force to have the english link as a type requirement. We only have a unique function to "translate" links, which I think is important for consistency.